routes/contactUsRoute.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import * as authentication from '../../middlewares/middlewares-babel/authenticate.js'
+import * as contact from '../../controllers/controller-babel/contactUsController';
```

Comment:
why would you have a folder named `middlewares-babel` and `controller-babel`

Reply:
that's where my babbled js is

Reply:
Any folder  with my js I babel all js for production and then place them in that folder

Reply:
I have actually found the way to babel without compiling them and storing them, I will create a branch after working on all tasks and then apply it